src/_settings.scss

```diff
@@ -0,0 +1,763 @@
+$scally-g-font-size-heading-x-large: $scally-g-font-size-heading + 4 !default;
+
+$scally-g-font-size-heading-xx-large: $scally-g-font-size-heading + 6 !default;
+
+$scally-g-font-size-heading-xxx-large: $scally-g-font-size-heading + 8 !default;
+
+
+/**
+ * Line height.
+ */
+
+$scally-g-line-height: $scally-g-font-size * 1.5 !default;
+
+
+/**
+ * Line height ratio.
+ *
+ * N.B. non-overridable.
+ */
+
+$scally-g-line-height-ratio: $scally-g-line-height / $scally-g-font-size;
+
+
+/**
+ * Families.
+ */
+
+// Base
+$scally-g-font-family-base: sans-serif !default;
+
+// Headings
+$scally-g-font-family-headings: $scally-g-font-family-base !default;
+
+// Mono-space
+$scally-g-font-family-monospace: Consolas, monospace !default;
+
+
+/**
+ * Weights.
+ *
+ * N.B. all the font weights are there to be used when you're using webfonts
+ * i.e. fonts brought in via the `@font-face` declaration. It's better to
+ * store each webfont weights in settings rather than trying to remember each
+ * of the numbers that the weights correspond to, plus it ensures that your
+ * project's typography is applied consistency. You can use these settings for
+ * none webfonts i.e. system fonts but typically you'll only need to apply one
+ * weight which is **bold** and in that case using `font-weight: bold;` is
+ * fine.
+ */
+
+$scally-g-font-weight-normal: 400 !default;
+
+$scally-g-font-weight-thin: 100 !default;
+
+$scally-g-font-weight-extra-light: 200 !default;
+
+$scally-g-font-weight-light: 300 !default;
+
+$scally-g-font-weight-medium: 500 !default;
+
+$scally-g-font-weight-semi-bold: 600 !default;
+
+$scally-g-font-weight-bold: 700 !default;
+
+$scally-g-font-weight-extra-bold: 800 !default;
+
+$scally-g-font-weight-black: 900 !default;
+
+// Heading weight
+$scally-g-font-weight-heading: $scally-g-font-weight-bold !default;
+
+
+/**
+ * Maximum line length using the `ch` unit.
+ */
+
+$scally-g-max-line-length: 600ch !default;
+
+
+
+
+/* 3. Spacing (Global)
+   ========================================================================= */
+
+/**
+ * The base spacing unit uses the base line height to achieve a vertical
+ * rhythm.
+ */
+
+$scally-g-spacing-base: $scally-g-line-height !default;
+
+// Decrease
+$scally-g-spacing-small: floor($scally-g-spacing-base / 1.5) !default;
+
+$scally-g-spacing-x-small: floor($scally-g-spacing-base / 2) !default;
+
+$scally-g-spacing-xx-small: floor($scally-g-spacing-base / 3) !default;
+
+$scally-g-spacing-xxx-small: floor($scally-g-spacing-base / 4) !default;
+
+$scally-g-spacing-xxxx-small: floor($scally-g-spacing-base / 6) !default;
+
+// Increase
+$scally-g-spacing-large: ceil($scally-g-spacing-base * 1.3) !default;
+
+$scally-g-spacing-x-large: ceil($scally-g-spacing-base * 1.5) !default;
+
+$scally-g-spacing-xx-large: ceil($scally-g-spacing-base * 1.7) !default;
+
+$scally-g-spacing-xxx-large: ceil($scally-g-spacing-base * 2) !default;
+
+$scally-g-spacing-xxxx-large: ceil($scally-g-spacing-base * 2.2) !default;
+
```

Comment:
I felt it gave the best coverage up and down.

Here are the values based on the default **24px** `line-height` (**16px** font size):

```scss
// Decrease
$scally-g-spacing-small: 16;

$scally-g-spacing-x-small: 12;

$scally-g-spacing-xx-small: 8;

$scally-g-spacing-xxx-small: 6;

$scally-g-spacing-xxxx-small: 4;

// Increase
$scally-g-spacing-large: 32;

$scally-g-spacing-x-large: 36;

$scally-g-spacing-xx-large: 41;

$scally-g-spacing-xxx-large: 48;

$scally-g-spacing-xxxx-large: 53;
```